React/Modules/RCTAccessibilityManager.m

```diff
@@ -162,6 +163,16 @@ - (void)setMultipliers:(NSDictionary<NSString *, NSNumber *> *)multipliers
   self.multipliers = multipliers;
 }
 
+RCT_EXPORT_METHOD(setAccessibilityFocus:(nonnull NSNumber *)reactTag)
+{
+  dispatch_async(RCTGetUIManagerQueue(), ^{
+    [self.bridge.uiManager addUIBlock:^(__unused RCTUIManager *uiManager, NSDictionary<NSNumber *, UIView *> *viewRegistry) {
```

Comment:
This will make it so that we'll wait for the next UI batch to complete. If you don't want that, dispatch directly to the main queue and use RCTUIManager's viewForTag method.